subprojects/docs/src/docs/userguide/command_line_interface.adoc

```diff
@@ -340,6 +340,9 @@ Sets maximum number of workers that Gradle may use. _Default is number of proces
 `--parallel`, `--no-parallel`::
 Build projects in parallel. For limitations of this option, see <<multi_project_builds.adoc#sec:parallel_execution, Parallel Project Execution>>. _Default is off_.
 
+`--priority`::
+Specifies the scheduling priority for the Gradle daemon and all processes launched by it. Values are `normal` or `low`. _Default is normal_.
```

Comment:
nit:
```
_Default is `normal`_.
```

Reply:
I didn't do this because it renders as verbatim backticks instead of coloring like in the previous sentence.

Reply:
Alright. 🙂